sds/src/secondary_validation/mod.rs

```diff
@@ -173,6 +175,9 @@ impl Validator for SecondaryValidator {
             SecondaryValidator::ItalianNationalIdChecksum => {
                 ItalianNationalIdChecksum.is_valid_match(regex_match)
             }
+            SecondaryValidator::JwtClaimsChecker { config } => {
+                JwtClaimsChecker::new(config.clone()).is_valid_match(regex_match)
```

Comment:
This is cloning the config and creating a new `JwtClaimsChecker` every time the validator runs. The main problem is right now the code assumes that the "Config" that is directly deserialized is the validator that will run since all validators until now have been lightweight / don't need any compilation. We will either need to change this a bit to allow a "compilation" step, or do a more short term fix here to cache the compilation. But right now this is still compiling the regex / other setup for every validation call which is way too expensive.